app/flatpak-builtins-run.c

```diff
@@ -85,6 +87,12 @@ static GOptionEntry options[] = {
   { "parent-expose-pids", 0, 0, G_OPTION_ARG_NONE, &opt_parent_expose_pids, N_("Make processes visible in parent namespace"), NULL },
   { "parent-share-pids", 0, 0, G_OPTION_ARG_NONE, &opt_parent_share_pids, N_("Share process ID namespace with parent"), NULL },
   { "instance-id-fd", 0, 0, G_OPTION_ARG_INT, &opt_instance_id_fd, N_("Write the instance ID to the given file descriptor"), NULL },
+  { "flatpak-internal-dev-shm-path", '\0',
+    G_OPTION_FLAG_HIDDEN, G_OPTION_ARG_FILENAME, &opt_dev_shm_path,
+    "Reuse this directory for subsandbox's /dev/shm", "PATH" },
+  { "flatpak-internal-tmp-path", '\0',
+    G_OPTION_FLAG_HIDDEN, G_OPTION_ARG_FILENAME, &opt_tmp_path,
+    "Reuse this directory for subsandbox's /tmp", "PATH" },
```

Comment:
I haven't made these part of the documented CLI because I can't see why anything other than a sandbox would legitimately use them.

Reply:
These have gone away.